website/pages/docs/k8s/installation/index.mdx

```diff
@@ -108,13 +95,27 @@ elected and every node will have a running Consul agent.
 
 If you want to customize your installation,
 create a `config.yaml` file to override the default settings.
-You can learn what settings are available by running `helm inspect values ./consul-helm`
+You can learn what settings are available by running `helm inspect values hashicorp/consul`
 or by reading the [Helm Chart Reference](/docs/platform/k8s/helm).
 
+We recommend setting the `global.name` value to `"consul"` so resources
+don't have the release name prefixed:
+
+```yaml
+# config.yaml
+global:
+  name: consul
+```
```

Comment:
Should we use a different example for customization since we're already using `--set global.name=consul` above in the `Installing Consul` section. Maybe enable sync and connect would be a good example?